docs/guide/execute-transaction.md

```diff
@@ -4,8 +4,7 @@ layout: splash
 
 # Execute Transaction
 
-`executeTx` is a high level method of Deployer which can be used to perform transactions on Algorand Network. It supports every transaction (atomic or single) which is possible in network. Ex: Deploy ASA/App, Opt-In, Transfers, Delete, Destroy etc. `executeTx` takes `ExecParams[]` as parameter.
-If the length of `ExecParams` array is greater than one, it will be considered as `atomic transaction`.
+`executeTx` is a high level method of Deployer which can be used to perform transactions on Algorand Network. It supports every transaction (atomic or single) which is possible in network. Ex: Deploy ASA/App, Opt-In, Transfers, Delete, Destroy etc. `executeTx` takes `ExecParams[]` or `SignedTransaction[]` as parameter. If the length of `ExecParams` array is greater than one, it will be considered as `atomic transaction` otherwise `single transaction`.
```

Comment:
Lets add a section where we shortly describe `ExecParams` and `SignedTransaction`. The differences and why would one prefer to use one over another

Reply:
I have created a section for ExecParams and we already have a section for SignedTransaction. I have also highlighted the difference between them.